tuning/config/acceleration_configs/fast_moe.py

```diff
@@ -42,8 +44,15 @@
 @parsable_dataclass
 @dataclass
 class FastMoe:
+    ep_degree: Union[int, bool] = 1
```

Comment:
do we not need to specify this is a `field` which can take both values

Reply:
Went with the previous coding style for dataclasses write for fms-acceleration plugins. I think we can have it as is WDYT?